### PR TITLE
For #19820: Specify usage of labels for a11y services.

### PR DESCRIPTION
**Pull request by mcarare**
### Pull Request checklist

- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not -> No tests needed for comments
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not -> No visual changes.
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by mcarare** [comment]
[taskcluster:error] Task timeout after 7200 seconds. Force killing container.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1547,9 +1547,9 @@
     <string name="credit_cards_card_number">Card Number</string>
     <!-- The header for the expiration date of a credit card -->
     <string name="credit_cards_expiration_date">Expiration Date</string>
-    <!-- The label for the expiration date month of a credit card -->
+    <!-- The label for the expiration date month of a credit card to be used by a11 services-->
```
Comment (Delphine) [inline app/src/main/res/values/strings.xml:1550]: Nit: a11y

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1547,9 +1547,9 @@
...
     <string name="credit_cards_expiration_date_month">Expiration Date Month</string>
-    <!-- The label for the expiration date year of a credit card -->
+    <!-- The label for the expiration date year of a credit card to be used by a11 services-->
```
Comment (Delphine) [inline app/src/main/res/values/strings.xml:1552]: Same nit: a11y

**Review by Delphine — CHANGES_REQUESTED** [review]

**Review by Delphine — APPROVED** [review]
Thank you! :)

**Pull request merged**